### ISSUE TITLE
time

### ISSUE THREAD

**Issue by connectuum** [connectuum/tenput#1]
`[...]`

**Comment by connectuum** [comment]
### 2021.12.31 `2h`

`-------` wrote most of the notes

(commit notes)
this was after highlighting on [the _Red Book_](https://connectuum.github.io/redbook/index.html), which ended midway through "Descent into Hell" with an idea-image in commit https://github.com/connectuum/redbook/commit/584d4da2f757365d8b50d14c432d58eecb8919ec. then i continued to read _Neverwhere_ for an hour or so. i seem to remember page 132 or 134 (not sure) but at some point an idea hit me about pushing out _ideas_ into GitHub projects, i can't possibly take on all these projects in time -- and also an odder image: pulling feature branches from other iterations of myself (who had already done the work in parallel). so i set the book aside and started to write.

**Comment by connectuum** [comment]
### 2022.01.01 `2.5h`

https://github.com/connectuum/tenput/commit/30b4f7224bc24b0983c77755bfb09e00ea1c3608 initial commit
https://github.com/connectuum/tenput/commit/539f5a0e24f622ba5707bee18f909fa54d1fa97d announced the project seed (combined and revised notes)
https://github.com/connectuum/tenput/commit/ed3327a232987ae84325ad9e98afe6ef5a0b1ea0 revised the alarm
